src/js/cards.js

```diff
@@ -18,8 +18,7 @@ export const DEFAULT_ATTACK_RATE = 3000;
      randomTarget   - BOOL           whether the target(s) are chosen randomly
      damage         - INT            deals damage to target(s)
      kill           - BOOL           kills permanents when used
-     description    - STRING         what the card does (TODO: this should actually be auto-genned)
-     flavor         - STRING         text just for fun 
+     description    - STRING ARRAY   what the card does (TODO: this should actually be auto-genned)
```

Comment:
Yes, minor point but it didn't seem necessary to have an additional key for that object, seems like two parts of a description to me.